web/frontend/package.json

```diff
@@ -56,6 +56,7 @@
     "copy-prime-assets": "node ./scripts/copy-prime-assets.js",
     "start": "node ./node_modules/vite/bin/vite.js --host",
     "build": "node ./node_modules/vite/bin/vite.js build && npm run copy-prime-assets",
+    "build-prod": "node ./node_modules/vite/bin/vite.js build --no-sourcemap && npm run copy-prime-assets",
```

Comment:
This prevents the building of control.js.map which saves many megabytes of output that would normally be embedded in the binary.